File: poc/InkPoc/InkPoc/Services/Ink/EventHandlers/AddStrokeEventArgs.cs
using System;
using Windows.UI.Input.Inking;

namespace InkPoc.Services.Ink
{
    public class AddStrokeEventArgs : EventArgs
    {
        public InkStroke OldStroke { get; set; }
        public InkStroke NewStroke { get; set; }

        public AddStrokeEventArgs(InkStroke newStroke, InkStroke oldStroke = null)
        {
            NewStroke = newStroke;
            OldStroke = oldStroke;
        }
    }
}

File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServicesVB/EventHandlers/Ink/AddStrokeEventArgs.vb
Imports Windows.UI.Input.Inking

Namespace EventHandlers.Ink
    Public Class AddStrokeEventArgs
        Inherits EventArgs

        Public Property OldStroke As InkStroke

        Public Property NewStroke As InkStroke

        Public Sub New(newStroke As InkStroke, Optional oldStroke As InkStroke = Nothing)
            NewStroke = newStroke
            OldStroke = oldStroke
        End Sub
    End Class
End Namespace
